### PR TITLE
TPROD-224: Confirmme theme back into mautic/mautic

### PR DESCRIPTION
**Pull request by shinde-rahul**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2.

### CODE REVIEW

**Comment by nickveenhof** [comment]
Shouldn't we at least fix the folder typo? confimme vs confirmme

**Comment by RCheesley** [comment]
@shinde-rahul could we fix the typo as mentioned above - it should indeed be 'confirmme' and was an oversight in the original PR!

**Comment by nickveenhof** [comment]
Also, please add a composer.json file in there, see https://github.com/mautic/theme-confirmme/blob/main/composer.json

File: themes/confirmme/html/message.html.twig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% extends ":"~template~":base.html.twig" %}
+{% block stylesheets %}
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" type="text/css" />
```
Comment (nickveenhof) [inline themes/confirmme/html/message.html.twig:3]: This is maybe a bit nitpicking, but shouldn't we use a bootstrap version that is supported today? Eg, bootstrap v5. For such a small theme, that change seems minimal?

Comment (RCheesley) [reply themes/confirmme/html/message.html.twig:3]: @jos0405 one for you!

**Review by RCheesley — APPROVED** [review]
Thanks for making the changes @shinde-rahul I just had to make one fix as we have changed the way that we specify the supported builders to

`"builder": ["grapesjsbuilder"],`

Otherwise this works great.

**Comment by RCheesley** [comment]
@all-contributors please add @jos0405 for code

**Comment by RCheesley** [comment]
@all-contributors please add @nickveenhof for review

**Pull request merged**